css/organization-page.styl

```diff
@@ -101,10 +101,20 @@
     &:hover
       background: TEAL_LIGHT
 
+  &__links
+    display: flex
+    flex-direction: column
+
   &__link
+    align-items: center
+    display: flex
     font-size: 1.2em
-    list-style: none
-    margin-bottom: 1.2em
+    margin-left: .8em
+    margin-top: 20px
+    text-decoration: none
+
+  &__icon
+    padding-right: .1em
```

Comment:
I think this solves the icon spacing issue, though there may be a more elegant solution